contracts/ClaimSettlment.sol

```diff
@@ -5,9 +5,15 @@ import "@openzeppelin/contracts/interfaces/IERC721.sol";
 import "@openzeppelin/contracts/interfaces/IERC20.sol";
 import "@gnosis.pm/zodiac/contracts/core/Module.sol";
 import "./ClaimSettlementBase.sol";
+import "@openzeppelin/contracts-upgradeable/utils/structs/EnumerableSetUpgradeable.sol";
 
 contract ClaimSettlement is ClaimSettlementBase {
+    using EnumerableSetUpgradeable for EnumerableSetUpgradeable.AddressSet;
     address public keySigner;
```

Comment:
We can remove this var now too

Reply:
I think as a result recommened we still need to using `EnumerableSet`

Reply:
https://github.com/cardstack/safe-module-claim-settlement/blob/28c4db43a2253c6539c508c55f41c29fa9187296/contracts/ClaimSettlment.sol#L11